schemas/iso19139/src/main/plugin/iso19139/schematron/schematron-rules-iso.sch

```diff
@@ -348,21 +348,19 @@ USA.
 			<sch:report test="gmd:aggregateDataSetName or gmd:aggregateDataSetIdentifier"
 				>$loc/strings/report.M23</sch:report>
 
-
-
-      <sch:let name="mdRefRef" value="gmd:aggregateDataSetIdentifier/*/gmd:code/*"/>
+      <sch:let name="mdRefRef" value="gmd:aggregateDataSetIdentifier/*/gmd:code/gco:CharacterString"/>
```

Comment:
> $mdRefRef contains two elements: gco:CharacterString and geonet:element

Forgot about that case with editing info.

You can try with:

```suggestion
      <sch:let name="mdRefRef" value="gmd:aggregateDataSetIdentifier/*/gmd:code/(gco:CharacterString|gmx:Anchor)"/>
```

Reply:
I have added the gmx:Anchor option and included a unit test to check the various combinations.